### PR TITLE
option renaming and upgrades

### PR DESCRIPTION
**Pull request by sophieberkhout**
Classical and Bayesian meta-analysis ([#2019](https://github.com/jasp-stats/INTERNAL-jasp/issues/2019), [#2014](https://github.com/jasp-stats/INTERNAL-jasp/issues/2014)).

I did not rename options with `visible:		module == "cochrane"`, because I assume that would break things in the cochrane module. @FBartos not sure if other options being renamed is a problem for that module.

### CODE REVIEW

**Comment by sophieberkhout** [comment]
Oops forgot to update tests.. one sec.

**Comment by sophieberkhout** [comment]
So the remaining problem with the unit tests (only for macOS) seems to be with the robust meta-analysis which I made no changes to.

I think this is the problem:
> fatal error: /Applications/Xcode_13.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo: can't open input file: /usr/local/libexec/jags-terminal (No such file or directory)

@boutinb you mentioned in the slack something about not finding the jags package. Could this be related?

File: inst/Upgrades.qml
Comment (Kucharssim) [inline inst/Upgrades.qml:12]: could you replace spaces with tabs in the qml files please?

File: inst/Upgrades.qml
Comment (Kucharssim) [inline inst/Upgrades.qml:9]: given we'll merge this after this release, you can change it to `fromVersion: "0.16.4"` and `toVersion: "0.17"`

File: inst/qml/qml_components/BayesianMetaAnalysisPriors.qml
Comment (Kucharssim) [inline inst/qml/qml_components/BayesianMetaAnalysisPriors.qml:88]:
```suggestion
					name: 			"normalSd"
```

**Review by Kucharssim — CHANGES_REQUESTED** [review]
Looks quite good, just a couple of minor comments.

> I did not rename options with visible: module == "cochrane", because I assume that would break things in the cochrane module. @FBartos not sure if other options being renamed is a problem for that module.

That should not be a problem for that module as currently `jaspCochrane` does not source the QML files directly but relies on copies of the original files from here, so you should not be able to break anything. So it would make more sense to do the changes here and the renaming in the Cochrane module would be done just by @FBartos sourcing the files directly and copying the relevant sections for the Upgrader once this PR is merged..

**Comment by Kucharssim** [comment]
Could you [rebase](https://github.com/jasp-stats/jasp-desktop/blob/stable/Docs/development/git-guide.md#rebasing-your-repository-github-documentation-rebasing) your branch and resolve the conflicts? Let me know if you have any help with that!

**Pull request closed without merging**
